html/core/src/jsMain/kotlin/org/jetbrains/compose/web/css/StyleSheet.kt

```diff
@@ -38,12 +41,21 @@ class CSSRulesHolderState : CSSRulesHolder {
  * ```
  */
 open class StyleSheet(
+    prefix: String?,
     private val rulesHolder: CSSRulesHolder = CSSRulesHolderState(),
-    val usePrefix: Boolean = true,
 ) : StyleSheetBuilder, CSSRulesHolder by rulesHolder {
     private val boundClasses = mutableMapOf<String, CSSRuleDeclarationList>()
+    val prefix = prefix ?: "${this::class.simpleName}-"
```

Comment:
does `prefix` need to be public? can it be private?

Also to preserve the compatibility, maybe it's worth to add a public `val usePrefix` with only getter that returns true when prefix is not null. 

I gues `val usePrefix` didn't need to be public at all, but since it's already here, it's better to avoid breaking changes.

Reply:
> to see it in master and in next update of compose
it will likely go into 1.6

Reply:
Looks like I resolved this point

Reply:
Done :)

Reply:
@eymar :)